### PR TITLE
Migrate some benchmarks to NNBD

### PR DESCRIPTION
**Pull request by goderbauer**
https://github.com/flutter/flutter/issues/71505

### CODE REVIEW

File: dev/benchmarks/platform_views_layout/test_driver/scroll_perf_test.dart [hunk 1]
```diff
@@ -7,17 +7,17 @@ import 'package:test/test.dart' hide TypeMatcher, isInstanceOf;
 
 void main() {
   group('scrolling performance test', () {
-    FlutterDriver driver;
+    FlutterDriver? driver;
```
Comment (dnfield) [inline dev/benchmarks/platform_views_layout/test_driver/scroll_perf_test.dart:10]: nit: late

File: dev/benchmarks/platform_views_layout_hybrid_composition/test_driver/scroll_perf_test.dart [hunk 2]
```diff
@@ -7,17 +7,17 @@ import 'package:test/test.dart' hide TypeMatcher, isInstanceOf;
 
 void main() {
   group('scrolling performance test', () {
-    FlutterDriver driver;
+    FlutterDriver? driver;
```
Comment (dnfield) [inline dev/benchmarks/platform_views_layout_hybrid_composition/test_driver/scroll_perf_test.dart:10]: ditto

**Review by dnfield — APPROVED** [review]
LGTM with nits

File: dev/benchmarks/platform_views_layout/test_driver/scroll_perf_test.dart (hunk 1 above)
Comment (goderbauer) [reply dev/benchmarks/platform_views_layout/test_driver/scroll_perf_test.dart:10]: The tear-down is doing a null-check before closing the driver. We'd have to remove that and assume that the driver is never null if we make it late here.

Comment (dnfield) [reply dev/benchmarks/platform_views_layout/test_driver/scroll_perf_test.dart:10]: `connect` can't return null with NNBD.

The only problem you could run into is connect throwing, in which case I don't think it even matters if we're in a bad state by tear down.

**Pull request merged**